### PR TITLE
Verifying that web methods may be defined as default interface methods

### PR DESCRIPTION
**Pull request by jglick**
Relevant since Jenkins core is now on `-source 8`.

@reviewbybees

### CODE REVIEW

**Comment by jglick** [comment]
CI build is getting a SIGKILL for some reason.

**Comment by jglick** [comment]
In fact [this build passed](https://jenkins.ci.cloudbees.com/job/core/job/stapler/178/) so it is just odd that there seem to be two jobs building the same repo. I will disable the other!

File: pom.xml [hunk 1]
```diff
@@ -83,7 +83,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <java.level>7</java.level>
+    <java.level>8</java.level>
```
Comment (oleg-nenashev) [inline pom.xml]: Requires a decision by Stapler maintainers. LGTM, but may require extra backporting methods if somebody wants to just update LTS version to the Master branch. I don't

Comment (jglick) [inline pom.xml]: > Requires a decision by Stapler maintainers.

Well, I cut the last release…

> if somebody wants to just update LTS version to the Master branch

I think generally we do not do that. Rather, we would create a backport branch and cherry-pick selected fixes.

**Comment by jglick** [comment]
@reviewbybees done

**Comment by jglick** [comment]
but CC @kohsuke in case he has any opinion

**Comment by jglick** [comment]
Will hold off on merge just in case.

**Pull request merged**